module/zfs/spa_misc.c

```diff
@@ -3135,7 +3188,6 @@ EXPORT_SYMBOL(spa_has_slogs);
 EXPORT_SYMBOL(spa_is_root);
 EXPORT_SYMBOL(spa_writeable);
 EXPORT_SYMBOL(spa_mode);
-EXPORT_SYMBOL(spa_namespace_lock);
```

Comment:
I'm not sure if we need to export something else to replace this. I searched the Lustre repos and its not referenced there, and I don't know who else might be using it, so I didn't bother for now. Easy enough to add later, but if someone wants to let me know its easy to add here too.